libraries/GCS_MAVLink/GCS.h

```diff
@@ -28,6 +28,7 @@
 #include <AP_AHRS/AP_AHRS_config.h>
 #include <AP_Arming/AP_Arming_config.h>
 #include <AP_Airspeed/AP_Airspeed_config.h>
+#include <AP_Scheduler/AP_Scheduler.h>
```

Comment:
This include needs to be removed.

It introduces a lot of dependencies, and since a *lot* of places use GCS.h ths will slow builds down.